tox.ini

```diff
@@ -283,6 +283,7 @@ commands =
 
 # Global settings
 
+disallow_incomplete_defs = True
```

Comment:
No incomplete definitions left; we can enable this check now.

Reply:
:tada: